ci-operator/step-registry/openstack/conf/addetcdramdisk/openstack-conf-addetcdramdisk-commands.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+
+IGNITIONVERSION="3.1.0"
```

Comment:
This might cause problems. If the step is used, there might be issues with the older versions of openshift-install which use lower ignition versions. 
Can it be left out?

Reply:
I am fairly certain that you cannot omit the Ignition version from the MachineConfig. I do not know which version of Ignition are supported by which version of OpenShift beyond knowing that OpenShift 4.6 does not support any 2.x Ignition versions.

Reply:
Moreover, using MachineConfigs may not even work for you as you want it to in earlier versions. If I understand it correctly, it is a fairly recent change to MCO that adds support for MachineConfigs added in the installer manifests to be included in the initial Ignition config supplied to the masters.

Reply:
According to manuals it si supported as as of 4.3,  I think perhaps we can at least figure out what version we are running and decide on doing something or not.

Reply:
After manual testing, on 4.3, 4.4, and 4.5, 4.6 and 4.7  I can verify that using IGNITIONVERSION=2.2.0 works for all cases, as expected. 
Rebasing patch to make just one commit